examples/step-90/step-90.cc

```diff
@@ -612,10 +612,10 @@ namespace Step90
                       pcout,
                       TimerOutput::never,
                       TimerOutput::wall_times)
-    , fe_degree(1)
+    , fe_degree(2)
     , fe_collection(FE_Q<dim>(fe_degree), FE_Nothing<dim>())
     , dof_handler(triangulation)
-    , level_set_fe_degree(1)
+    , level_set_fe_degree(2)
```

Comment:
Please leave these as linear elements, so as not to change the output of the tutorial.